ee/tables/execparsers/key_value/test-data/nmcli.txt

```diff
@@ -0,0 +1,663 @@
+﻿NAME:                                                                      AP[1]
```

Comment:
I think we should remove this test data and replace with something anonymized

Reply:
Scamble the BSSID and SSID values